Initial WG LTS Reformation (#7287)
kubernetes/community: wg-lts/README.md

<!---
This is an autogenerated file!

Please do not edit this file directly, but instead make changes to the
sigs.yaml file in the project root.

To understand how this file is generated, see https://git.k8s.io/community/generator/README.md
--->
# LTS Working Group

> The working group is organized with the goal of developing a better understanding of what "Long Term Support" might mean for Kubernetes, those who support  Kubernetes, and end users. The working group will investigate changes the Kubernetes project could make related this better understanding of what long term support might mean. The working group will also determine the feasibility, benefits, **cost**, and prerequisites  of any such changes.s

Reviewer: ```suggestion
The working group is organized with the goal of developing a better understanding of what "Long Term Support" might mean for Kubernetes, those who support Kubernetes, and end users. The working group will investigate changes the Kubernetes project could make related to the better understanding of what long term support might mean. The working group will also determine the feasibility, benefits, **cost**, and prerequisites  of any such changes.
```

> The working group is organized with the goal of developing a better understanding of what "Long Term Support" might mean for Kubernetes, those who support  Kubernetes, and end users. The working group will investigate changes the Kubernetes project could make related to the better understanding of what long term support might mean. The working group will also determine the feasibility, benefits, **cost**, and prerequisites  of any such changes.

Reviewer 2: ```suggestion
The working group is organized with the goal of developing a better understanding of what "Long Term Support" might mean for Kubernetes, those who support Kubernetes, and end users. The working group will investigate changes the Kubernetes project could make related to the better understanding of what long term support might mean. The working group will also determine the feasibility, benefits, **cost**, and prerequisites of any such changes.
```
